libraries/AP_HAL_Linux/Util_RPI.cpp

```diff
@@ -33,15 +33,15 @@ int UtilRPI::_check_rpi_version()
     hw = Util::from(hal.util)->get_hw_arm32();
 
     if (hw == UTIL_HARDWARE_RPI2) {
-        printf("Raspberry Pi 2 with BCM2709!\n");
+        printf("Raspberry Pi 2/3 with BCM2709!\n");
         _rpi_version = 2;
     } else if (hw == UTIL_HARDWARE_RPI1) {
         printf("Raspberry Pi 1 with BCM2708!\n");
         _rpi_version = 1;
     } else {
         /* defaults to 1 */
```

Comment:
Sorry for catching this just now, but it defaults to 2 now, right?
Well, anyway, I think this comment should be just removed, since that pretty clear in the code :-)